xsl/pmo/options.xsl

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016-2018 Zerocracy
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to read
+the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+merge, publish, distribute, sublicense, and/or sell copies of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns="http://www.w3.org/1999/xhtml" version="1.0">
+  <xsl:include href="../templates.xsl"/>
+  <xsl:template match="/">
+    <html lang="en">
+      <body>
+        <section>
+          <h1>
+            <xsl:text>Options</xsl:text>
+          </h1>
+          <xsl:apply-templates select="options"/>
+        </section>
+      </body>
+    </html>
+  </xsl:template>
+  <xsl:template match="options">
+    <p>
+      <xsl:text>You can download and upload options file here</xsl:text>
+      <!--
+        @todo #258:30min Add some controls to allow user to download and
```

Comment:
@g4s8 this is not relevant to this repository. The functionality will be implemented in "farm". Remove this puzzle. This XSL will only render the options. Uploading will happen in another page.